internal/controller/providers/psmdb/applier.go

```diff
@@ -282,8 +283,9 @@ func (p *applier) exposeDefaultReplSet(expose *psmdbv1.ExposeTogglable) error {
 	database := p.DB
 	switch database.Spec.Proxy.Expose.Type {
 	case everestv1alpha1.ExposeTypeInternal:
-		expose.Enabled = false
+		expose.Enabled = true
```

Comment:
Is this a mistake?

Reply:
no, it is intended. the expose should be enabled, it's just the type has changed. I noticed that when we switch from LoadBalancer to ClusterIP, the actual service type in k8s was never changed, and this was what fixed it, now the type is changed as expected.